velox/docs/functions/spark/misc.rst

```diff
@@ -15,11 +15,21 @@ Miscellaneous Functions
         SELECT at_least_n_non_nulls(2, 0, 1.0, NULL);  -- true
         SELECT at_least_n_non_nulls(2, 0, array(NULL), NULL);  -- true
 
+.. spark:function:: get_array_struct_fields(array, ordinal) -> array(T)
+
+    Extracts the ``ordinal``-th fields of all array elements, and returns them as a new array.
+    The first input must be of array(strcut) type and nested complex type is allowed.
+    The ``ordinal`` is 0-based, and if ``ordinal`` is negative or greater than or equal to
+    the children size of strcut, exception is thrown. ::
+
+        SELECT items.col1 FROM VALUES (array(struct(100,'foo'), struct(200,'bar'))) AS t(items); -- array(100, 200)
+        SELECT items.col2 FROM VALUES (array(struct(100,'foo'), struct(200,'bar'))) AS t(items); -- array('foo', 'bar')
+
 .. spark:function:: get_struct_field(struct, ordinal) -> T
 
     Returns the value of nested subfield at position ``ordinal`` in the input ``struct``.
     The input must be of row type and nested complex type is allowed.
-    The ``ordinal`` is 0-based, and if ``ordinal`` is negative or greater than or equal to
+    The ``ordinal`` is 0-based, and if ``ordinal`` is negative or no less than
```

Comment:
nit: seems modified another function, and the `get_array_struct_fields` was not updated  :)